lib/graphql/query/executor.rb

```diff
@@ -13,7 +13,9 @@ def initialize(query)
       # @return [Hash] A GraphQL response, with either a "data" key or an "errors" key
       def result
         execute
-      rescue GraphQL::Query::OperationNameMissingError, GraphQL::Query::VariableMissingError => err
+      rescue GraphQL::ExecutionError => err
```

Comment:
:clap: :clap:  good call to add a proper exception hierarchy. Maybe `OperationNameMissingError` could be refactored down the road, too